src/main/Main.hs

```diff
@@ -631,8 +631,8 @@ cleanCmd opts go =
   -- See issues #2010 and #3468 for why "stack clean --full" is not used
   -- within docker.
```

Comment:
So interestingly, both #2010 and #3468 have been closed — but #3471 is still open.

@mgsloan can this be simplified, now?